testproj/requirements.txt

```diff
@@ -0,0 +1,13 @@
+coreapi>=2.3.3
+coreschema>=0.0.4
+Django==2.0
```

Comment:
Django is intentionally not listed in requirements since it is expected that people installing a **Django library** already have Django.

Reply:
I believe this expectation is not correct. I believe it is considered to be a bad practice to have packages installed globally, including Django